portfolio/src/main/java/com/google/sps/servlets/FavShowServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import com.google.gson.Gson;
```

Comment:
Google's Java Style prefers sorting of imports: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

I believe this would make com.google.gson.Gson first.

Reply:
Done